### DISCUSSION TITLE
Calibrating the pension system

### DISCUSSION THREAD

**Discussion by jdebacker** [EAPD-DRB/OG-PHL#19] [General]
A resource with a good summary of how social security retirement benefits are calculated: https://www.sss.gov.ph/sss/appmanager/viewArticle.jsp?page=retirement

**Comment by ptuano** [comment]
Just remember that the Social Security System webpage calculator is for private sector retirees; the [Government Service Insurance System](https://www.gsis.gov.ph/ginhawa-for-all/retirement-benefit/) has a different sector for public sector employees.
